src/main/java/com/socrata/api/SodaDdl.java

```diff
@@ -355,16 +352,9 @@ public ClientResponse issueRequest() throws LongRunningQueryException, SodaError
 
         try {
             final ClientResponse response = requester.issueRequest();
-            return mapper.readValue(response.getEntity(InputStream.class), AssetResponse.class);
-            //return response.getEntity(AssetResponse.class);
+            return response.getEntity(AssetResponse.class);
```

Comment:
I cannot find any documentation for the "assets" resource of the Import API. Does `response.getEntity` not work because this resource does not accept requests for `application/json` so the `MessageBodyReader` will not be triggered?